osf/models/brand.py

```diff
@@ -1,16 +1,17 @@
 from osf.models.base import BaseModel
 from django.db import models
-from colorfield.fields import ColorField
 
 
 class Brand(BaseModel):
     """
     This model holds data custom styled assets for providers
     """
 
+    name = models.CharField(max_length=30, blank=True, null=True)
```

Comment:
I don't think this should have a name, it's basically this is only ever going to be associated with the Registration. As much as I'd love to have brand names like "California Policy Lab Blue" or "Carnegie Mellon Classic". Though I could be persuaded if we're are actually keeping around unused brands. What does Abram/Brian thing about this?

Reply:
I included this so it would be easier to select from brands in the admin app. Brands will be created, then you will select the brand to associate in the registration provider (or other provider in the future). The name will make it easier to differentiate between brands than just having a sequential ID. This won't be visible to the end user, or exposed in the API. Its just for the admin app.

Reply:
Yeah visibility on the admin app is reason enough. But we still need to remember to give them cool names.